scripts/daily_upload_s3.coffee

```diff
@@ -51,6 +51,6 @@ db.articles.find({ published: true })
       'Content-Type': 'text/csv'
     }, (err, result) ->
 
-      # Delete filename and close db
+      # Delete filename and exit
       fs.unlink(dir + filename)
-      db.close()
+      process.exit(1)
```

Comment:
This should just be `process.exit()`—add the 1 indicates it's exiting with an error, which in this case we want to indicate an intentional exit.